zebrad/tests/common/lightwalletd/send_transaction_test.rs

```diff
@@ -95,6 +95,40 @@ pub async fn run() -> Result<()> {
         assert_eq!(response, expected_response);
     }
 
+    // We test mempool grpc calls here as we know we have mempool transactions
+
+    // Transactions can be in the RPC queue, lets wait until they are retired
+    // before checking the mempool.
+    tokio::time::sleep(tokio::time::Duration::from_secs(120)).await;
+
+    // Call `GetMempoolStream`
+    let mut response = rpc_client
+        .get_mempool_stream(wallet_grpc::Empty {})
+        .await?
+        .into_inner();
+
+    // Make sure transactions are returned from the mmepool
+    let mut counter = 0;
+    while let Some(_txs) = response.message().await? {
+        counter += 1;
+    }
+    assert!(counter > 0);
```

Comment:
This is weird, it works locally but fails in the CI : https://github.com/ZcashFoundation/zebra/runs/6670629763?check_suite_focus=true#step:9:707

Reply:
I look into this a bit and don't know what's happening... the logs seen OK. The "REORG" messages from lightwalletd are a bit weird, but it manages to sync correctly. I'll get back at this in a while.

Reply:
@oxarbitrage can you please sync this PR with `main` so that we have another go at CI with the recent changes from `main`?

Reply:
> The "REORG" messages from lightwalletd are a bit weird, but it manages to sync correctly.

We expect about 100 of these when `zebrad` restarts, because it discards the non-finalized blocks. (`lightwalletd` treats tip changes as a reorg, unless they're a child of the previous tip.)

Reply:
> @oxarbitrage can you please sync this PR with `main` so that we have another go at CI with the recent changes from `main`?

I'm not sure that will help, it seems like `main` is still using `adityapk00/lightwalletd`, despite PR #4553:
> {"app":"lightwalletd","error":"error parsing block: parsing transaction 0: while parsing transparent input: could not read ScriptSig","height":1687104,"level":"warning","msg":"error zcashd getblock rpc","time":"2022-06-04T08:33:12Z"}

https://github.com/ZcashFoundation/zebra/runs/6736917351?check_suite_focus=true#step:9:1154

But this works for me locally when I run tests with https://github.com/zcash/lightwalletd

Maybe @gustavovalverde can tell us why this repository build isn't working:
https://github.com/ZcashFoundation/zebra/blob/00aa64a387ec0dca68cfaf139bfd4374c23b3c7b/.github/workflows/zcash-lightwalletd.yml#L53-L54

Reply:
I also re-opened ticket https://github.com/ZcashFoundation/zebra/issues/4551